core/src/main/java/greencity/dto/ratingstatistics/RatingStatisticsDto.java

```diff
@@ -0,0 +1,20 @@
+package greencity.dto.ratingstatistics;
+
+import greencity.annotations.RatingCalculationEnum;
+import greencity.entity.User;
+import java.time.ZonedDateTime;
+import lombok.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
+public class RatingStatisticsDto {
+    private Long id;
+    private ZonedDateTime createDate;
+    private RatingCalculationEnum ratingCalculationEnum;
+    private float pointsChanged;
+    private float rating;
+    private User user;
```

Comment:
There will be a problem later. `RatingStatisticsDto` is a DTO from Controllers layer. `User` is an Entity from DAO layer. Please either replace by something like `String username` or introduce a kind of `UserDto`.